data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1951 @@
+			variant
+				"Heliarch Hunter"
+		system "Speloog"
+	npc evade
+		government "Heliarch"
+		personality heroic opportunistic target
+		fleet
+			names "heliarch"
+			variant
+				"Heliarch Hunter"
+		system "Torbab"
+	npc evade
+		government "Heliarch"
+		personality heroic opportunistic target
+		fleet
+			names "heliarch"
+			variant
+				"Heliarch Hunter"
+		system "Flugbu"
+	npc evade
+		government "Heliarch"
+		personality heroic opportunistic target
+		fleet
+			names "heliarch"
+			variant
+				"Heliarch Hunter"
+		system "Debrugt"
+	on visit
+		dialog `After you land, one of the Heliarch Hunters who was pursuing you lands beside you. You'll need to shake it off before landing here to succeed.`
+	on complete
+		"coalition jobs" ++
+		payment 470000
+		conversation
+			`Magister Lotuora meets you when you land, handing you your payment of <payment>, and congratulating you for managing to evade the pursuing ships. "Frustrated many of the Hunter pilots, I heard you have," she says with a laugh. "Good training for them, I hope this was. Finishing up the arrangements for one more drill, I am. When ready, you are, meet me in the spaceport, you must."`
+
+mission "Heliarch Drills 3"
+	name "Combat Drill"
+	description "Head to the <waypoints> system to fight the <npc>, and report back to <planet> when you succeed. You must only disable the ship. Destroying or capturing it would deem you an enemy of the Heliarchs."
+	source "Belug's Plunge"
+	waypoint "Torbab"
+	to offer
+		has "Heliarch Drills 2-C: done"
+		not "Lunarium: Questions: done"
+	on offer
+		conversation
+			`Lotuora is speaking with a Kimek Heliarch captain at the entrance to the spaceport. "Ah, there the Captain is," she tells the Kimek. "Captain <last>, your sparring partner, this is," she says, gesturing to the Kimek, who salutes you with a short bow of her head. "Excited to test her ship against an outsider, she is."`
+			`	The Kimek doesn't stay for long, only wishing you a "good fight" and heading to the hangars, where she boards a Punisher and flies away. You look to Lotuora, waiting for her to explain. "Tough ships, our Punishers are, but looked into your encounter records, we have, and experience in fighting, you have," she says. "To avoid any accidents, we want, so swapped out, the ship's Finisher Pods were. A mere drill, this is, so only aim to disable one another, you shall. If successful, you are, set aside <payment> for your efforts, we have."`
+			`	She stares on for a while, then brings up a map. "Headed to the <waypoints> system, the Punisher is. Wait for you there, it will." She tells you to head to <waypoints> when you're ready to fight the Punisher, and reminds you to only disable the ship, as any harsher action would make you a criminal to the Heliarchs.`
+				accept
+	npc disable save
+		government "Heliarch Test Dummy"
+		personality disables staying nemesis heroic
+		system "Torbab"
+		fleet
+			names "heliarch"
+			variant
+				"Heliarch Punisher (Scrappy)"
```

Comment:
I fully intend on having the Heliarch side start off easier but climb to much harder in terms of combat difficulty, as you go from fighting much weaker civilian ships to Quarg HWs. So, while upon beginning it will not warrant much, by the end you should be very well armed.
This mission set checks for CR >= 8104 before starting up at all; I could increase that to some other value if it's preferred.
The final mission of this chain is one without time constraints, with a single ship that's stuck in an uninhabited system that one can easily avoid jumping to. I don't see why the ship should be toned down considering that the duel with it is pretty much all the challenge for this mission set.
Will edit the convo to add her saying you can just go and tackle the ship whenever you feel you're ready, as there's no rush.